lib/skpm-build.js

```diff
@@ -132,9 +132,14 @@ function getCommands (manifestJSON) {
         process.exit(1)
       }
     }
+
     // always expose the default
-    prev[c.script].handlers.push('onRun')
-    prev[c.script].identifiers.push(c.identifier)
+    if (!prev[c.script].handlers.includes('onRun')) {
```

Comment:
can you use `indexOf` instead of `includes` (which requires nodejs >= 6 and I think we still support nodejs 4)?